Python/Product/Common/Strings.resx

```diff
@@ -2957,8 +2957,10 @@ Please specify at least one package.</value>
     <value>Discovery results file '{0}' was not found.</value>
   </data>
   <data name="ErrorTestContainerNotFound" xml:space="preserve">
-    <value>Test container was not found for {0}.</value>
-    <comment>{0} is the test info</comment>
+    <value>Test container was not found under folder {0} 
+for test {1}.</value>
+    <comment>{0} is the root directory
```

Comment:
maybe its better to not use a line break here?

Reply:
I'd vote for no line break. User can Toggle Word Wrap on the output window.

Reply:
ok. updating